cmd/ocm-backplane/cloud/assume.go

```diff
@@ -0,0 +1,140 @@
+package cloud
+
+import (
```

Comment:
A small one, could you sort a bit about the deps? Also in the other files as well.

Reply:
My IDE automatically sorts imports for me, and this is how it ended up. I also tried a `$ gofmt -w cmd/ocm-backplane/cloud/assume.go` and it left these imports as is. Is there something specific you're looking for here?

Reply:
ok, looks like there are different sort styles, my IDE uses `goimports` by default, I believe some of them are using `gofmt` by default. As we do not have a unified policy here. Let's leave it for now.